### PR TITLE
fix labels, disable Fluent-Bit Helm Chart serviceMonitor, use Prometheus Operator additionalServiceMonitors to work around expectation that Prometheus Operator is already installed.

### PR DESCRIPTION
**Pull request by frankreno**
fix labels, disable Fluent-Bit Helm Chart serviceMonitor, use Prometheus Operator additionalServiceMonitors to work around expectation that Prometheus Operator is already installed.

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -221,29 +217,43 @@ prometheus-operator:
   prometheus:
     additionalServiceMonitors:
       - name: collection-sumologic
-        additionalLabels: 
+        additionalLabels:
           app: sumologic
-          release: prometheus-operator
+          release: collection
```
Comment (lei-sumo) [inline deploy/helm/sumologic/values.yaml]: We will also need to document to remind user to override all these hard corded value if they specify a different release name cc @maimaisie

**Comment by rvmiller89** [comment]
@frankreno I get this error trying this, maybe I'm doing something wrong?
```
k8s $ k get pods
No resources found.
k8s $ k get servicemonitors
No resources found.
k8s $ helm install sumologic/sumologic --name collection --namespace sumologic -f values.yaml --no-crd-hook
Error: release collection failed: servicemonitors.monitoring.coreos.com "collection-fluent-bit" already exists
```

**Comment by rvmiller89** [comment]
Frank and I figured out the issue I was having with `"collection-fluent-bit" already exists` ...

The problem is that I was applying the new `values.yaml` over the repository helm chart, which still had the old fluent-bit service monitor declaration, so it was trying to declare the same service monitor twice.  This will be resolved once we update the repository version.

File: deploy/helm/prometheus-overrides.yaml [hunk 2]
```diff
@@ -6,29 +6,43 @@ grafana:
...
-      - port: metrics
+        - port: metrics
       namespaceSelector:
         matchNames:
-        - sumologic
+          - sumologic
+      selector:
+        matchLabels:
+          app: fluentd-events
+    - name: collection-fluent-bit
```
Comment (maimaisie) [inline deploy/helm/prometheus-overrides.yaml]: @frankreno this file is generated from `values.yaml`. is this `name: collection-fluent-bit` correct for the `prometheus-overrides.yaml` file?

**Pull request merged**